replication/parser.go

```diff
@@ -100,7 +100,10 @@ func (p *BinlogParser) ParseReader(r io.Reader, onEvent OnEventFunc) error {
 		var e Event
 		e, err = p.parseEvent(h, data)
 		if err != nil {
-			break
+			if _, ok := err.(missingTableMapEvent); ok {
```

Comment:
I went back and forth on this internally before just picking a way so that a patch could be submitted for discussion about it (indecision being the enemy of progress...)

It's really a matter of complexity and where that complexity is 

- It's complex to configure
- It's complex to catch
- It's complex to debug

vs surprise

- I'm surprised that it worked
- I'm surprised that it didn't work

Since it's your package I'm happy to go however you want on this. But the current method of seeming like it worked because it didn't return an error and also not doing *anything at all* is more complex and surprising than anything we choose here :)